src/uct/api/v2/uct_v2.h

```diff
@@ -549,43 +555,49 @@ ucs_status_t uct_md_mem_dereg_v2(uct_md_h md,
  */
 typedef enum uct_md_attr_field {
     /** Indicate max allocation size. */
-    UCT_MD_ATTR_FIELD_MAX_ALLOC        = UCS_BIT(0),
+    UCT_MD_ATTR_FIELD_MAX_ALLOC                 = UCS_BIT(0),
 
     /** Indicate max registration size. */
-    UCT_MD_ATTR_FIELD_MAX_REG          = UCS_BIT(1),
+    UCT_MD_ATTR_FIELD_MAX_REG                   = UCS_BIT(1),
 
     /** Indicate capability flags. */
-    UCT_MD_ATTR_FIELD_FLAGS            = UCS_BIT(2),
+    UCT_MD_ATTR_FIELD_FLAGS                     = UCS_BIT(2),
 
     /** Indicate memory types that the MD can register. */
-    UCT_MD_ATTR_FIELD_REG_MEM_TYPES    = UCS_BIT(3),
+    UCT_MD_ATTR_FIELD_REG_MEM_TYPES             = UCS_BIT(3),
 
     /** Indicate memory types that the MD can register. */
-    UCT_MD_ATTR_FIELD_CACHE_MEM_TYPES  = UCS_BIT(4),
+    UCT_MD_ATTR_FIELD_CACHE_MEM_TYPES           = UCS_BIT(4),
 
     /** Indicate memory types that the MD can detect. */
-    UCT_MD_ATTR_FIELD_DETECT_MEM_TYPES = UCS_BIT(5),
+    UCT_MD_ATTR_FIELD_DETECT_MEM_TYPES          = UCS_BIT(5),
 
     /** Indicate memory types that the MD can allocate. */
-    UCT_MD_ATTR_FIELD_ALLOC_MEM_TYPES  = UCS_BIT(6),
+    UCT_MD_ATTR_FIELD_ALLOC_MEM_TYPES           = UCS_BIT(6),
 
     /** Indicate memory types that the MD can access. */
-    UCT_MD_ATTR_FIELD_ACCESS_MEM_TYPES = UCS_BIT(7),
+    UCT_MD_ATTR_FIELD_ACCESS_MEM_TYPES          = UCS_BIT(7),
 
     /** Indicate memory types for which the MD can return a dmabuf_fd. */
-    UCT_MD_ATTR_FIELD_DMABUF_MEM_TYPES = UCS_BIT(8),
+    UCT_MD_ATTR_FIELD_DMABUF_MEM_TYPES          = UCS_BIT(8),
 
     /** Indicate registration cost. */
-    UCT_MD_ATTR_FIELD_REG_COST         = UCS_BIT(9),
+    UCT_MD_ATTR_FIELD_REG_COST                  = UCS_BIT(9),
 
     /** Indicate component name. */
-    UCT_MD_ATTR_FIELD_COMPONENT_NAME   = UCS_BIT(10),
+    UCT_MD_ATTR_FIELD_COMPONENT_NAME            = UCS_BIT(10),
+
+    /** Indicate size of buffer needed for packed rkey. */
+    UCT_MD_ATTR_FIELD_RKEY_PACKED_SIZE          = UCS_BIT(11),
+
+    /** Indicate CPUs near the resource. */
+    UCT_MD_ATTR_FIELD_LOCAL_CPUS                = UCS_BIT(12),
 
     /** Indicate size of buffer needed for packed rkey. */
-    UCT_MD_ATTR_FIELD_RKEY_PACKED_SIZE = UCS_BIT(11),
+    UCT_MD_ATTR_FIELD_EXPORTED_MKEY_PACKED_SIZE = UCS_BIT(13),
```

Comment:
line 596 - change packed to exported

Reply:
also the comment is copy-pasted from `UCT_MD_ATTR_FIELD_RKEY_PACKED_SIZE`